ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -26,6 +26,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      customLLDBInitFile = "$(SRCROOT)/Flutter/ephemeral/flutter_lldbinit"
```

Comment:
Looking at this line more closely, the content of it sounds like something the `flutter` tool might put there.

And indeed, `git log -G flutter_lldbinit` in the Flutter tree finds this from earlier this month, 2025-03-06:
flutter/flutter@eb07c51230c70a0ecaf010a02e7862f69be3b32b

So this is a side effect of the Flutter upgrade in #1426.

Reply:
Right, I'd only ran those commits in the draft (#1422) on Android, sorry, forgot to note that in #1426.

(Guess this could've been caught if we had the iOS build in CI (#773), which I think should be good to have especially now, as #1379 introduces some Swift code.)

Reply:
Cool, makes sense.

And yeah, agreed on both points re #773.

Reply:
Made a note of that on #773, and bumped it up to M5b (from M6).